docs/troubleshoots.md

```diff
@@ -0,0 +1,28 @@
+# TROUBLESHOOTS 
```

Comment:
```suggestion
# Troubleshoots
```
My suggestion is opinionated and based on the other markdown files in the repo. For consistency can you make it like this?

Reply:
ok will update the file now 😉